src/components/MessageList/VirtualizedMessageList.tsx

```diff
@@ -179,21 +179,39 @@ const VirtualizedMessageListWithContext = <
     setNewMessagesNotification(false);
   }, [virtuoso, processedMessages, setNewMessagesNotification, processedMessages.length]);
 
+  const [newMessagesReceivedInBackground, setNewMessagesReceivedInBackground] = React.useState(
+    false,
+  );
+
+  const resetNewMessagesReceivedInBackground = useCallback(() => {
+    setNewMessagesReceivedInBackground(false);
+  }, [setNewMessagesReceivedInBackground]);
```

Comment:
```suggestion
  }, []);
```
You can omit this one, `setState` functions are "stable" and won't change.